.github/workflows/e2e-nvidia-t4-x1.yml

```
@@ -15,6 +17,14 @@ on:
     branches:
       - main
       - release-*
+    paths:
+      # note this should match the merging criteria in 'mergify.yml'
+      - '**.py'
+      - 'pyproject.toml'
+      - 'requirements*.txt'
```

```suggestion
      - 'requirements**.txt'
```

To match regex in mergify.